package.json

```diff
@@ -1,16 +1,48 @@
 {
   "private": true,
-  "workspaces": [
-    "dataregistry",
-    "dataregistry-extension"
-  ],
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ]
+  },
   "scripts": {
-    "build": "tsc --build dataregistry dataregistry-extension && jlpm workspace @jupyterlab/dataregistry-extension run generateSchema",
-    "build:dev": "jlpm install && jlpm clean && jlpm run build && jlpm run link",
-    "lint":  "jlpm workspace @jupyterlab/dataregistry-extension lint",
-    "clean": "jlpm workspace @jupyterlab/dataregistry run clean && jlpm workspace @jupyterlab/dataregistry-extension run clean",
-    "build:watch": "tsc --build --watch  dataregistry dataregistry-extension",
-    "link": "jupyter labextension link dataregistry --no-build && jupyter labextension link dataregistry-extension",
-    "unlink": "jupyter labextension unlink dataregistry --no-build && jupyter labextension unlink dataregistry-extension"
+    "all": "jlpm run clean && jlpm run build && jlpm run build:jupyter",
+    "all:watch": "jlpm run clean && jlpm run build && jlpm run build:jupyter:watch",
+    "build": "jlpm run build:dev",
+    "build:dev": "jlpm install && jlpm run clean:packages && lerna link && jlpm run build:packages && jlpm run link:packages",
```

Comment:
Any reason  to use `lerna link` over `lerna bootstrap`? I don't know the difference but the lerna readme just mentions bootstrap.

Reply:
Yeah, did not want Lerna executing any of the other commands in `bootstrap` beside linking.

Reply:
See [here](https://github.com/lerna/lerna/tree/master/commands/bootstrap#usage).

Reply:
Ah thank you! Sounds good